.lgtm

```diff
@@ -0,0 +1,2 @@
+pattern = "(?)LGTM"
```

Comment:
I have set up a basic pattern; This is just for a start. This will accept any string that ends with LGTM.